.github/workflows/deploy-prod.yml

```
@@ -0,0 +1,53 @@
+name: ReLink Production CI/CD
+
+on:
+  push:
+    branches: [ "main", "dev" ] # main, dev 브랜치 직접 푸시 및 타 브랜치 PR 머지(Merge) 완료 시 실행
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)